neuralcoref/train/conllparser.py

```diff
@@ -315,7 +313,9 @@ def add_conll_utterance(self, parsed, tokens, corefs, speaker_id, use_gold_menti
                 self.mentions.append(mention)
                 #            print("mention: ", mention, "label", mention.gold_label)
         else:
-            self._extract_mentions(parsed, len(self.utterances), self.n_sents, self.speakers[speaker_id])
+            mentions_spans = extract_mentions_spans(doc=parsed, blacklist=self.blacklist)
```

Comment:
Check if you want to use it (what's in blacklist)

Reply:
Otherwise looks good to me

Reply:
it's not used by default, I made it an argparse parameter to be able to set it easily